files/usr/bin/cinnamon-install-spice

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/python3
+
+import sys
+import os
+import argparse
+
+import gi
+gi.require_version('Gtk', '3.0')
```

Comment:
You don't seem to be using the gi import in this file.

Reply:
Those lines prevent a warning when Spices.py is imported.

Reply:
I see, then, shouldn't these lines be added to Spices.py?

Reply:
Probably. They weren't there before because Spices.py was always only imported from cinnamon-settings, which already had those lines. Another option could be to see if we can refactor it so that we don't need to use Gtk as an import in Spices.py.